_data/feedback.yml

```diff
@@ -0,0 +1,2 @@
+# replace this with the Google Forms URL once it is ready.
+https://pysangamam.org
```

Comment:
probably an overkill but felt I will have it ready just in case I am not available at night.